File: aquality-appium-java-master/src/test/java/base/BaseTest.java
package base;

import aquality.appium.application.ApplicationManager;
import io.appium.java_client.android.AndroidDriver;
import org.testng.annotations.AfterClass;
import org.testng.annotations.BeforeClass;

public class BaseTest {

    private AndroidDriver<?> driver;

    @BeforeClass
    public void setUp() {
        driver = (AndroidDriver<?>) ApplicationManager.getApplication().getDriver();
    }

    @AfterClass
    public void tearDown() {
        ApplicationManager.getApplication().quit();
    }
}

File: KotlinSecretCloset/src/test/kotlin/base/BaseTest.kt
package base

import framework.application.App
import org.testng.annotations.AfterClass
import org.testng.annotations.BeforeClass

open class BaseTest {

    @BeforeClass
    open fun setUp() {
        App.setUpDriver()
    }

    @AfterClass
    open fun tearDown() {
        App.quit()
    }
}